cmd/root/auth.go

```diff
@@ -195,6 +195,12 @@ func MustWorkspaceClient(cmd *cobra.Command, args []string) error {
 		cfg.Profile = profile
 	}
 
+	_, isTargetFlagSet := targetFlagValue(cmd)
+	if !isTargetFlagSet && hasProfileFlag {
+		// If the profile flag is set but the target flag is not, we should skip loading the bundle configuration.
+		cmd.SetContext(SkipLoadBundle(cmd.Context()))
```

Comment:
Does skip affect DABs function?

Can I do `databricks bundle validate -t prod -p myprofile`? If yes, do we have a test for that?